service.go

```diff
@@ -131,6 +135,48 @@ func FindServiceAndAdd(user *gumble.User, url string) error {
 	}
 }
 
+// FindServiceAndSearch tries to find the right service and gives the url escaped query to it.
+// The resulting string is a URL to the video/song and its supplied to the function FindServiceAndAdd
+func FindServiceAndSearch(user *gumble.User, searchString string) error {
+	var searchService Service
+
+	var serviceProvider, argument string
+	split := strings.Split(searchString, "\n")
+	splitString := split[0]
+	if strings.Contains(splitString, " ") {
+		index := strings.Index(splitString, " ")
+		serviceProvider, argument = splitString[0:index], splitString[(index+1):]
```

Comment:
`serviceProvider` should be converted to lowercase here to ensure that a match occurs when different capitalization is used in the search command.